modules/olm-disabling-copied-csvs.adoc

```diff
@@ -12,6 +12,19 @@ When Operators are configured to use the `AllNamespaces` install mode, versus ta
 
 To support these larger clusters, cluster administrators can disable copied CSVs for Operators installed with the `AllNamespaces` mode.
 
+[WARNING]
+====
+If you disable copied CSVs, a user's ability to discover Operators in the OperatorHub and CLI is limited to Operators that meet the following criteria:
+* The Operator is configured to reconcile events in the user's namespace.
```

Comment:
Is this true? I had thought that the user would only discover CSVs of operators installed in the namespace.

Reply:
I misunderstood. I thought it had to be installed directly in the namespace and be configured to reconcile events in the namespace.

Reply:
@awgreene I updated the content based on your comment. Would you PTAL?